examples/package.json

```diff
@@ -18,7 +18,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "bignumber.js": "^9.0.0",
+    "bn.js": "^5.0.0",
```

Comment:
I'm curious why we're moving away from `bignumber.js` to `bn.js`? 🤔

Reply:
consistency.. web3 also converting to `bn.js`
https://web3js.readthedocs.io/en/v1.2.0/web3-utils.html#tobn

we also check for this instance, so should use bn everywhere